File: Majo/Act-Instructor_Henry/lenguajes/ruby/junio/17-06-24_2.rb
puts "Cantidad de aprendices:"
n = gets.chomp.to_i

for x in 1..n
  puts "Introduce tu nombre:"
  nm = gets.chomp
  a = nm.length - 1
  nm_inv = ""
  for i in (a..0).step(-1)
    s = nm[i]
    nm_inv += s
  end
  puts("tu nombre es: " + nm + " y la forma inversa es: " + nm_inv)
end

File: Majo/Act-Instructor_Henry/lenguajes/dart/junio/17-06-24_2.dart
import 'dart:io';

void main(){
  print("Cantidad de aprendices:");
  int n = int.parse(stdin.readLineSync()!);

  for (var x = 1; x <= n; x++) {
    print("Introduce tu nombre:");
    String? nm = stdin.readLineSync();

    int a = nm!.length;
    String nm_inv = "";

    for (var i = a - 1 ; i >= 0; i--) {
      String s = nm[i];
      nm_inv += s;
    }
  
  print("tu nombre es: " + nm + " y la forma inversa es: " + nm_inv);
  }


}